### PR TITLE
[Application] Bugfix in LLaMA Application

### PR DESCRIPTION
**Pull request by EunjuYang**
- This commit fixes a bug in `applyTKP` function.
- It seems applying Top-K and Top-P to `logits` didn't work as intended

Self evaluation:

Build test: [X]Passed [ ]Failed [ ]Skipped
Run test: [X]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

**Review by skykongkong8 — APPROVED** [review]
Nice catch!

**Review by myungjoo — APPROVED** [review]

**Review by djeong20 — APPROVED** [review]
LGTM

**Review by lhs8928 — APPROVED** [review]
LGTM

**Review by DonghakPark — APPROVED** [review]
LGTM!

**Review by SeoHyungjun — APPROVED** [review]
LGTM
